packages/admin/src/components/notification-banner/DownloadMessage.tsx

```diff
@@ -0,0 +1,36 @@
+type DownloadMessageProps = {
+  count: number;
+};
+
+const DownloadMessage = ({ count }: DownloadMessageProps) => {
+  return (
+    <div
+      className="govuk-notification-banner"
+      role="region"
+      aria-labelledby="govuk-notification-banner-title"
+      data-module="govuk-notification-banner"
+      data-testid="spotlight-banner"
+    >
+      <div className="govuk-notification-banner__header">
+        <h2
+          className="govuk-notification-banner__title"
+          id="govuk-notification-banner-title"
+        >
+          Important
+        </h2>
+      </div>
+      <div className="govuk-notification-banner__content">
+        <p className="govuk-notification-banner__heading">
+          Cannot download {count} applications
```

Comment:
Can we add logic here so it only adds the S to applications if count is bigger than 1? Just so that it reads better

Reply:
Changed in latest push!